PencilCase/PencilCase/scripts/application.ts

```diff
@@ -56,6 +56,17 @@ export class Application {
     private onDeviceReady = (): void => {
         document.addEventListener('pause', this.onPause, false);
         document.addEventListener('resume', this.onResume, false);
+        document.addEventListener('backbutton', function (evt) {
```

Comment:
Currently set backbutton event handler as do nothing, or it will cause some unexpected behavior.
Need to add one goBack method for each page to behavior the expected operation. So this method should be one members of the base page class.